.drone.star

```diff
@@ -18,7 +18,7 @@ OC_CI_PHP = "owncloudci/php:%s"
 OC_CI_WAIT_FOR = "owncloudci/wait-for:latest"
 OC_CS3_API_VALIDATOR = "owncloud/cs3api-validator:0.2.1"
 OC_LITMUS = "owncloudci/litmus:latest"
-OC_OC_TEST_MIDDLEWARE = "owncloud/owncloud-test-middleware:1.8.8"
+OC_OC_TEST_MIDDLEWARE = "owncloud/owncloud-test-middleware:latest"
```

Comment:
I would suggest to leave it pinned to a particular version

Reply:
https://github.com/owncloud/owncloud-test-middleware/pull/143#issuecomment-2014516613